docs/source/aws.rst

```diff
@@ -0,0 +1,38 @@
+
+===========================
+How to ingest data from AWS
+===========================
+1. Add AWS credentials to docker-compose
+
+Docker-compose has two repeated lines:
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+
+You'll need to set those environment variables with your AWS access and secret key. And then you can call docker-compose up
+
+=========================
+How to Create a Provider
+=========================
+Send a POST request to
+`http://<your_host>:8000/api/cost-management/v1/providers/`
```

Comment:
For RST inline literals like this, you want double backticks. See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup

Alternatively, if you want block code, you'll need it indented by four spaces with a blank line _both_ before and after in. See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks


```suggestion
```